golem-ts-agentic/packages/golem-ts-sdk/tests/invoke.test.ts

```diff
@@ -145,6 +145,33 @@ test('WeatherAgent can be successfully initiated and the methods can be invoked'
           resolvedAgent,
           `Weather in ${locationValue} is sunny!`,
         );
+
+        testInvoke(
+          typeRegistry,
+          'fun4',
+          'location',
+          { data: locationValue, value: number },
+          resolvedAgent,
+          undefined,
```

Comment:
The return type is undefined if we inspect the return value of a function that's `Promise<void>` or when user don't specify anything as return type